tutorials/run-laravel-on-appengine-standard/index.md

```diff
@@ -0,0 +1,259 @@
+---
+title: Run Laravel on Google App Engine Standard
+description: Learn how to deploy a Laravel app to Google App Engine standard environment.
+author: bshaffer
+tags: App Engine, Laravel, PHP
+date_published: 2019-01-15
+---
+## Laravel
+
+[Laravel][laravel] is an open source web framework for PHP developers that encourages the use of the model-view-controller (MVC) pattern.
+
+You can check out [PHP on Google Cloud Platform][php-gcp] (GCP) to get an
+overview of PHP and learn ways to run PHP apps on GCP.
+
+## Prerequisites
+
+1. Create a project in the [Google Cloud Platform Console](https://console.cloud.google.com/project).
+1. Enable billing for your project.
+1. Install the [Google Cloud SDK][cloud_sdk].
+
+> All code for this tutorial is available in the [PHP samples repository][laravel-framework-sample].
+
+## Prepare
+
+Follow the official documentation for [installing Laravel][laravel-install]
+from laravel.com.
+
+## Run
+
+1. Run the app with the following command:
+
+        php artisan serve
+
+1. Visit [http://localhost:8000](http://localhost:8000) to see the Laravel
+   Welcome page.
+
+## Deploy
+
+1. Create an `app.yaml` file with the following contents:
+
```

Comment:
Our publishing process doesn't currently deal well with code fences inside lists, so I'll need to switch this codeblock to just use indentation. I can do this during the production pass, so there's nothing required from you here now. I'm just letting you know what to expect.